docs/topics/db/models.txt

```diff
@@ -954,16 +954,43 @@ extend the parent's :ref:`Meta <meta-options>` class, it can subclass it. For ex
 
 Django does make one adjustment to the :ref:`Meta <meta-options>` class of an abstract base
 class: before installing the :ref:`Meta <meta-options>` attribute, it sets ``abstract=False``.
-This means that children of abstract base classes don't automatically become
-abstract classes themselves. Of course, you can make an abstract base class
-that inherits from another abstract base class. You just need to remember to
-explicitly set ``abstract=True`` each time.
+This means that children of abstract base classes don't automatically become abstract classes
+themselves. You can make an abstract base class that inherits from another abstract base
+class; you just need to remember to explicitly set ``abstract=True`` each time.
 
 Some attributes won't make sense to include in the :ref:`Meta <meta-options>` class of an
 abstract base class. For example, including ``db_table`` would mean that all
 the child classes (the ones that don't specify their own :ref:`Meta <meta-options>`) would use
 the same database table, which is almost certainly not what you want.
 
+Due to the way Python inheritance works, if a child class inherits from multiple abstract base
+classes, only the :ref:`Meta <meta-options>` options from the first listed class will be
+inherited by default. To inherit :ref:`Meta <meta-options>` options from multiple abstract
+base classes, or from a base class not listed first, you must explicitly declare the
```

Comment:
Feels like `or from a base class not listed first` is redundant here?

Reply:
Maybe I should reword, but if you only want to inherit from the second parent’s meta, you can do so if you’re explicit. Also maybe it’s over complicating here. Interested in other’s thoughts.

Reply:
I think it's over-complicating and can drop it. If the reader understands the "from multiple", they should be able to figure out the implication for inheriting from non-first parents.

Reply:
Sounds good - removed.